index.d.ts

```diff
@@ -516,6 +516,21 @@ declare module "react-native-maps" {
 
   export class Heatmap extends React.Component<MapHeatmapProps, any> {}
 
+  // =======================================================================
+  //  Geojson
+  // =======================================================================
+
+  import GeoJSON from 'geojson';
```

Comment:
needs a dependency on `@types/geojson`, I'm getting errors in CI after this change

![image](https://user-images.githubusercontent.com/1404810/78113092-8301a800-73ff-11ea-9d60-2de9d3ae72d1.png)

Reply:
I can't reproduce this on my project, can you tell me if simply adding `@types/geojson` to `devDependencies` fixes it? Could it also need module `geojson` itself?

Reply:
that would fix it, yeah. I don't use `react-native-maps` at all in my project, I get it via `@types/expo`

Reply:
so I'd like to not add `geojson` either.

If you `import` something, whether it's types or not, you should always have a dependency on it in your own package.json. Maybe a peer dep, but probably a regular dependency